css/index.css

```diff
@@ -98,15 +194,114 @@ p {
 .middle-content .boxes {
     display: flex;
     flex-wrap: wrap;
-    justify-content: space-evenly;
+    justify-content: space-around;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
+    width: 10;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
-    color: white;
+    color: black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.teal {
+    background: teal;
+    width: 12.5%;
+    height: 12.5%;
+    margin: 20px 2.5%;
```

Comment:
wondering why you did a hard coded top / bottom margin , then percentage based left/right margin